plugin/ochttp/propagation/tracecontext/propagation.go

```diff
@@ -98,13 +104,26 @@ func (f *HTTPFormat) SpanContextFromRequest(req *http.Request) (sc trace.SpanCon
 	return sc, true
 }
 
+func getRequestHeader(req *http.Request, name string, commaSeparated bool) (hdr string, ok bool) {
+	ok = false
+	for k, v := range req.Header {
+		if strings.EqualFold(k, name) {
+			if commaSeparated || len(v) == 1 {
+				ok = true
+			}
+			return strings.Join(v, ","), ok
```

Comment:
Can you give an example when this case happens?

A header key might have more than one value if user in the process mutates the header. Do we care about these cases?

Reply:
https://tools.ietf.org/html/rfc7230#section-3.2.2

This is to make sure the ochttp tracecontext propagation could handle duplicated header names. It works for tracestate with either single key-value pair or multiple pairs.

```
traceparent: 00-12345678901234567890123456789012-1234567890123456-00
tracestate: rojo=1,congo=2
tracestate: foo=3,bar=4
tracestate: baz=5
```

After string.Join, the tracestate string would be rojo=1,congo=2,foo=3,bar=4,baz=5